client/web/src/batches/BatchChangesNavItem.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import { NavItem, NavLink } from '@sourcegraph/wildcard'
+import { NavItem, NavLink } from '@sourcegraph/web/src/nav'
```

Comment:
```suggestion
import { NavItem, NavLink } from '../nav'
```

We can just use the relative import here

Reply:
ok, I'll push this